.github/clusterloader2-release.yml

```
@@ -0,0 +1,35 @@
+name: Release ClusterLoader2
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - v*
+  release:
+    types: [published, edited]
+
+env:
+  IMAGE_REPO: ghcr.io/${{ github.repository_owner }}
```

I actually got into an issue for the same, `github.repository_owner` is `Azure` here with A as uppercase, however docker repo cannot contain uppercase. You can check #5 to see the fix.

```suggestion
  IMAGE_REPO: ghcr.io/${{ github.repository_owner.toLowerCase }}
```

---

fixed now